base/compiler/ssair/ir.jl

```diff
@@ -250,7 +250,7 @@ function setindex!(is::InstructionStream, newval::Instruction, idx::Int)
     is.flag[idx] = newval[:flag]
     return is
 end
-function setindex!(is::InstructionStream, newval::AnySSAValue, idx::Int)
+function setindex!(is::InstructionStream, newval::Union{AnySSAValue, Nothing}, idx::Int)
```

Comment:
This change seems unnecessary?

Reply:
This regressed at some point. It's the idiom for deleting and instruction so should work, but yes, it's mostly unrelated to this PR.